plasmapy/setup_package.py

```diff
@@ -0,0 +1,35 @@
+# script for astropy helpers to use to customize setup.
+from setuptools.extension import Extension
+
+# list of cython extensions in package
+# cython_extensions = [Extension("*", ["*.pyx"])]
+
+# list of c extensions in package
+# c_extensions = [Extension("*", ["*.c"])]
+
+# getting C and Cython extensions
+cython_testFile = 'plasmapy/physics/parameters_cython.pyx'
+c_testFile = 'plasmapy/physics/parameters_cython.c'
+
+# names for compiling the .c files to
+compileName = 'parameters_cython'
+#compileNamePath = 'plasmapy/physics/parameters_cython'
+compileNamePath = 'plasmapy.physics.parameters_cython'
+compileName_cython = 'plasmapy/physics/parameters_cython_pyx'
+compileName_c = 'plasmapy/physics/parameters_cython_c'
```

Comment:
Most of these names are unused. Is that intended?

Reply:
This was for testing how to get inplace binaries (in the correct directories), as setuptools was fickle when the binary name and the .pyx file name didn't match exactly.
To fix this I just resorted to explicitly cythonizing the extensions in `setup.py` rather than letting setuptools handle it implicitly.

All of this will eventually have to be cleaned up to allow for building from the distributed `.c` files rather than cythonizing for every build, which require a `USE_CYTHON` flag in `setup.py` [like this](http://docs.cython.org/en/latest/src/reference/compilation.html#distributing-cython-modules) I think.